Test fails on non-English locales

### Search first

- [X] I searched and no similar issues were found

### Description
```
[ERROR] Failures:
[ERROR]   CommandLineUtilsTest.getBanner:17 Condition not satisfied:

banner.contains("Starting Liquibase at")
|      |
|      false
####################################################
##   _     _             _ _                      ##
##  | |   (_)           (_) |                     ##
##  | |    _  __ _ _   _ _| |__   __ _ ___  ___   ##
##  | |   | |/ _` | | | | | '_ \ / _` / __|/ _ \  ##
##  | |___| | (_| | |_| | | |_) | (_| \__ \  __/  ##
##  \_____/_|\__, |\__,_|_|_.__/ \__,_|___/\___|  ##
##              | |                               ##
##              |_|                               ##
##                                                ##
##  Get documentation at docs.liquibase.com       ##
##  Get certified courses at learn.liquibase.com  ##
##                                                ##
####################################################
Starte Liquibase am 11:04:41 (Version [local build] #0, kompiliert am 2023-05-05 08:56+0000)

[ERROR]   CommandLineUtilsTest.getBanner with banner disabled:29 Condition not satisfied:

banner.contains("Starting Liquibase at")
|      |
|      false
Starte Liquibase am 11:04:41 (Version [local build] #0, kompiliert am 2023-05-05 08:56+0000)
```
### Steps To Reproduce

On a non-English (e. g. German) locale execute `mvn package`.

### Expected/Desired Behavior

Test succeeds.

### Liquibase Version

latest

### Database Vendor & Version

none

### Liquibase Integration

none

### Liquibase Extensions

none

### OS and/or Infrastructure Type/Provider

Windows 11

### Additional Context

Locale: DE_de

### Are you willing to submit a PR?

- [X] I'm willing to submit a PR (Thank you!)

--- a/liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/generateChangelog.test.groovy
+++ b/liquibase-integration-tests/src/test/resources/liquibase/extension/testing/command/generateChangelog.test.groovy
@@ -1,5 +1,7 @@
 package liquibase.extension.testing.command
 
+import static java.util.ResourceBundle.getBundle
+
 import liquibase.change.ColumnConfig
 import liquibase.change.ConstraintsConfig
 import liquibase.change.core.AddForeignKeyConstraintChange
@@ -154,7 +156,7 @@ Optional Args:
             cleanResources("changelog-test.xml")
         }
         expectedException = CommandValidationException.class
-        expectedExceptionMessage = "Output ChangeLogFile 'target/test-classes/changelog-test.xml' already exists!"
+        expectedExceptionMessage = getBundle("liquibase/i18n/liquibase-core").getString("changelogfile.already.exists").replace("%s", "target/test-classes/changelog-test.xml")
     }
 
     run "Filtering with includeObjects", {

--- a/liquibase-standard/src/test/groovy/liquibase/integration/commandline/CommandLineUtilsTest.groovy
+++ b/liquibase-standard/src/test/groovy/liquibase/integration/commandline/CommandLineUtilsTest.groovy
@@ -19,7 +19,7 @@ class CommandLineUtilsTest extends Specification {
         then:
         banner.contains("Get documentation at docs.liquibase.com")
         banner.contains(coreBundle.getString("starting.liquibase.at.timestamp").replace("%s", ""))
-        banner.toLowerCase().contains("version ")
+        banner.contains(coreBundle.getString("liquibase.version.builddate").replaceFirst("%s.*", ""))
     }
 
     def "getBanner with banner disabled"() {
@@ -31,6 +31,6 @@ class CommandLineUtilsTest extends Specification {
         then:
         !banner.contains("Get documentation at docs.liquibase.com")
         banner.contains(coreBundle.getString("starting.liquibase.at.timestamp").replace("%s", ""))
-        banner.toLowerCase().contains("version ")
+        banner.contains(coreBundle.getString("liquibase.version.builddate").replaceFirst("%s.*", ""))
     }
 }

--- a/liquibase-standard/src/test/groovy/liquibase/integration/commandline/CommandLineUtilsTest.groovy
+++ b/liquibase-standard/src/test/groovy/liquibase/integration/commandline/CommandLineUtilsTest.groovy
@@ -4,7 +4,11 @@ import liquibase.GlobalConfiguration
 import liquibase.Scope
 import spock.lang.Specification
 
+import static java.util.ResourceBundle.getBundle
+
 class CommandLineUtilsTest extends Specification {
+
+    private static ResourceBundle coreBundle = getBundle("liquibase/i18n/liquibase-core")
 
     def getBanner() {
         when:
@@ -14,8 +18,8 @@ class CommandLineUtilsTest extends Specification {
 
         then:
         banner.contains("Get documentation at docs.liquibase.com")
-        banner.contains("Starting Liquibase at");
-        banner.contains("version ")
+        banner.contains(coreBundle.getString("starting.liquibase.at.timestamp").replace("%s", ""))
+        banner.toLowerCase().contains("version ")
     }
 
     def "getBanner with banner disabled"() {
@@ -26,7 +30,7 @@ class CommandLineUtilsTest extends Specification {
 
         then:
         !banner.contains("Get documentation at docs.liquibase.com")
-        banner.contains("Starting Liquibase at");
-        banner.contains("version ")
+        banner.contains(coreBundle.getString("starting.liquibase.at.timestamp").replace("%s", ""))
+        banner.toLowerCase().contains("version ")
     }
 }

--- a/liquibase-standard/src/test/groovy/liquibase/util/csv/CSVReaderTest.groovy
+++ b/liquibase-standard/src/test/groovy/liquibase/util/csv/CSVReaderTest.groovy
@@ -38,7 +38,7 @@ class CSVReaderTest extends Specification {
 
         then:
         def e = thrown(CsvMalformedLineException)
-        e.message.startsWith("Unterminated quoted field at end of CSV line")
+        e.lineNumber == 2
 
         where:
         separator                   | quote                             | input                  | notes

